### PR TITLE
Add missing info to success page

### PR DESCRIPTION
**Pull request by joeauyeung**
## What does this PR do?

Add missing info to booking success page from Figma.

Screenshot from 2022-05-04 15-08-38

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Make a booking

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/components/booking/pages/BookingPage.tsx [hunk 1]
```diff
@@ -101,7 +101,7 @@ const BookingPage = ({
 
   const mutation = useMutation(createBooking, {
     onSuccess: async (responseData) => {
-      const { attendees, paymentUid } = responseData;
+      const { id, attendees, paymentUid } = responseData;
```
Comment (joeauyeung) [inline apps/web/components/booking/pages/BookingPage.tsx:104]: Need to grab attendees and additional notes once booking is created

File: apps/web/components/booking/pages/BookingPage.tsx [hunk 2]
```diff
@@ -135,6 +135,7 @@ const BookingPage = ({
           email: attendees[0].email,
           location,
           eventName: profile.eventName || "",
+          bookingId: id,
```
Comment (joeauyeung) [inline apps/web/components/booking/pages/BookingPage.tsx:138]: Pass booking id to success page

File: apps/web/pages/success.tsx [hunk 3]
```diff
@@ -279,14 +280,32 @@ export default function Success(props: inferSSRProps<typeof getServerSideProps>)
...
                               ({localStorage.getItem("timeOption.preferredTimeZone") || dayjs.tz.guess()})
                             </span>
                           </div>
+                          <div className="font-medium">{t("who")}</div>
+                          <div className="col-span-2">
+                            {bookingInfo?.user && (
+                              <div className="mb-3">
+                                <p>{bookingInfo.user.name}</p>
+                                <p className="text-bookinglight">{bookingInfo.user.email}</p>
+                              </div>
```
Comment (joeauyeung) [inline apps/web/pages/success.tsx:298]: `user` field of booking is seperate from attendees

File: apps/web/pages/success.tsx [hunk 4]
```diff
@@ -301,6 +320,14 @@ export default function Success(props: inferSSRProps<typeof getServerSideProps>)
                               </div>
                             </>
                           )}
+                          {bookingInfo?.description && (
+                            <>
+                              <div className="mt-6 font-medium">{t("additional_notes")}</div>
+                              <div className="col-span-2 mt-6 mb-6">
+                                <p>{bookingInfo.description}</p>
+                              </div>
+                            </>
```
Comment (joeauyeung) [inline apps/web/pages/success.tsx:329]: Only show notes if they were added to the booking

File: apps/web/pages/success.tsx [hunk 5]
```diff
@@ -515,6 +542,8 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
   const typeId = parseInt(asStringOrNull(context.query.type) ?? "");
   const typeSlug = asStringOrNull(context.query.eventSlug) ?? "15min";
   const dynamicEventName = asStringOrNull(context.query.eventName) ?? "";
+  const bookingId = parseInt(context.query.bookingId as string);
```
Comment (joeauyeung) [inline apps/web/pages/success.tsx:545]: Grab the bookingId from the query

File: apps/web/pages/success.tsx [hunk 6]
```diff
@@ -583,6 +612,27 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
...
+      },
+      attendees: {
+        select: {
+          name: true,
+          email: true,
+        },
+      },
+    },
+  });
+
```
Comment (joeauyeung) [inline apps/web/pages/success.tsx:635]: Only query from the db the necessary information needed

File: apps/web/pages/success.tsx [hunk 7]
```diff
@@ -591,6 +641,7 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       trpcState: ssr.dehydrate(),
       dynamicEventName,
       userHasSpaceBooking,
+      bookingInfo,
```
Comment (joeauyeung) [inline apps/web/pages/success.tsx:644]: Add to page props

**Review by ciaranha — COMMENTED** [review]
Looks good to me on the design side and fixes the bug!

**Review by zomars — APPROVED** [review]

**Pull request merged**